volatility/framework/plugins/windows/malfind.py

```diff
@@ -21,16 +21,20 @@ class Malfind(interfaces.plugins.PluginInterface):
     def get_requirements(cls):
         # Since we're calling the plugin, make sure we have the plugin's requirements
         return [
-            requirements.TranslationLayerRequirement(name = 'primary',
-                                                     description = 'Memory layer for the kernel',
-                                                     architectures = ["Intel32", "Intel64"]),
-            requirements.SymbolTableRequirement(name = "nt_symbols", description = "Windows kernel symbols"),
-            requirements.ListRequirement(name = 'pid',
-                                         element_type = int,
-                                         description = "Process IDs to include (all other processes are excluded)",
-                                         optional = True),
-            requirements.PluginRequirement(name = 'pslist', plugin = pslist.PsList, version = (1, 0, 0)),
-            requirements.PluginRequirement(name = 'vadinfo', plugin = vadinfo.VadInfo, version = (1, 0, 0)),
+            requirements.TranslationLayerRequirement(name='primary',
+                                                     description='Memory layer for the kernel',
+                                                     architectures=["Intel32", "Intel64"]),
+            requirements.SymbolTableRequirement(name="nt_symbols", description="Windows kernel symbols"),
+            requirements.ListRequirement(name='pid',
+                                         element_type=int,
+                                         description="Process IDs to include (all other processes are excluded)",
+                                         optional=True),
+            requirements.BooleanRequirement(name='dump',
+                                            description="Extract injected VADs",
+                                            default=False,
+                                            optional=True),
+            requirements.PluginRequirement(name='pslist', plugin=pslist.PsList, version=(1, 0, 0)),
```

Comment:
So `PluginRequirement` is now technically the old way of doing things, and this will work, but to save us having to change it all later, I'd prefer to use `requirements.VersionRequirement(name='pslist', component=pslist.PsList, version=(1,0,0))`.  It's functionally identical, but we'll be using VersionRequirement in the future because it's more generic (and allows us to version anything we want rather than just plugins).

Reply:
Happy to be overridden on this one too, if people feel it'd be better to have specific names here...  5:)